apps/app/app/(authed)/collectivite/[collectiviteId]/(acces-restreint)/tableau-de-bord/personnel/_components/filtered-fiche-by-module.tsx

```diff
@@ -55,6 +56,7 @@ const buildFilterSearchParameters = (module: ModuleFicheActionsSelect) => {
 export const FilteredFichesByModule = ({
   module,
   onFilterChange,
+  hideEditAction,
```

Comment:
plutôt mettre des booléens positifs ('visible' plutôt que 'hidden'')

```suggestion
  canBeEdited,
```